cmd/clusterctl/client/cluster/mover.go

```diff
@@ -366,7 +366,7 @@ func (o *objectMover) backup(graph *objectGraph, directory string) error {
 	moveSequence := getMoveSequence(graph)
 
 	// Save all objects group by group
-	log.Info("Saving files to %s", directory)
+	log.Info(fmt.Sprintf("Saving files to %s", directory))
```

Comment:
Does this logger not have `Infof`?

Reply:
For future reference, let's make sure that PR titles have more explicit information, like `Fix panic in logger not using string interpolation ....`

Reply:
Yup will do. The logger has no Infof func (it's the go-logr/logr.Logger interface)